conf/routes

```diff
@@ -244,6 +245,8 @@ POST   /v3/studies/init                @org.sagebionetworks.bridge.play.controll
 GET    /v3/studies/self                @org.sagebionetworks.bridge.play.controllers.StudyController.getCurrentStudy
 GET    /v3/studies/self/publicKey      @org.sagebionetworks.bridge.play.controllers.StudyController.getStudyPublicKeyAsPem
 GET    /v3/studies/self/emailStatus    @org.sagebionetworks.bridge.play.controllers.StudyController.getEmailStatus
+POST   /v3/studies/self/resendVerifyConsentNotificationEmail @org.sagebionetworks.bridge.play.controllers.StudyController.resendVerifyConsentNotificationEmail
+POST   /v3/studies/self/verifyConsentNotificationEmail @org.sagebionetworks.bridge.play.controllers.StudyController.verifyConsentNotificationEmail
```

Comment:
Possibly we could use a format that'll allow us to add the support email later:

/v3/studies/self/emails/verify?type=[support|consentNotification]
/v3/studies/self/emails/resendVerify?type=[support|consentNotification]

Reply:
Changed, to allow future additions to this. Note that in the code, we currently only support consentNotification.